### PR TITLE
Basic soak test for Akka Cluster

### PR DESCRIPTION
**Pull request by chbatey**
- Add cinnamon
- Enable lightbend protmethus scraping for console
- Every 10s ping an actor on every node from every node to establish
connections between every node

The test job doesn't do anything yet, just getting some basic timings for now

### CODE REVIEW

File: cluster-sharding/src/main/resources/common.conf [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+  prometheus {
+    exporters += http-server
+
+    http-server {
+      host = "0.0.0.0"
+      port = 9001
+    }
+  }
+}
```
Comment (patriknw) [inline cluster-sharding/src/main/resources/common.conf]: add newline at end of file

File: cluster-soak-tests/src/main/resources/logback.xml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<configuration>
+
+    <appender name="CONSOLE" class="ch.qos.logback.core.ConsoleAppender">
+        <target>System.out</target>
+        <encoder>
+            <pattern>%date{MM/dd HH:mm:ss} %-5level[%thread] %logger{1} - %m%n%xException</pattern>
```
Comment (patriknw) [inline cluster-soak-tests/src/main/resources/logback.xml]: might be useful to include `%X{sourceThread}`, `%X{akkaSource}`, `%X{sourceActorSystem}`
https://doc.akka.io/docs/akka/current/logging.html#logging-thread-akka-source-and-actor-system-in-mdc

What is that `%xException` at the end?

File: cluster-soak/src/main/resources/logback.xml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<configuration>
+
+    <appender name="CONSOLE" class="ch.qos.logback.core.ConsoleAppender">
+        <target>System.out</target>
+        <encoder>
+            <pattern>%date{MM/dd HH:mm:ss} %-5level[%thread] %logger{1} - %m%n%xException</pattern>
```
Comment (patriknw) [inline cluster-soak/src/main/resources/logback.xml]: ^

**Review by patriknw — APPROVED** [review]
LGTM

File: cluster-soak/src/main/resources/logback.xml (hunk 3 above)
Comment (chbatey) [reply cluster-soak/src/main/resources/logback.xml]: Created a common one they all share

File: cluster-soak-tests/src/main/resources/logback.xml (hunk 2 above)
Comment (chbatey) [reply cluster-soak-tests/src/main/resources/logback.xml]: :+1: added source/thread. system will be the same for all of them. Not sure where that Exception came from.

**Pull request merged**